### ISSUE TITLE
About execution of depth_integrator.py [Windows10]

### ISSUE THREAD

**Issue by tkuri** [mitsuba-renderer/mitsuba2#30]
Hello, sorry I have one more question.

**What I am doing**
I'm trying to run the sample code along the tutorial on this page.
https://mitsuba2.readthedocs.io/en/latest/src/python_interface/rendering_scene/#custom-rendering-pipeline-in-python
When I run `docs/examples/02_depth_integrator/depth_integrator.py`, the following error occurs.
```
2020-03-12 07:04:00 INFO  main  [xml.cpp:1117] Loading XML file "..\..\..\..\mitsuba-data\scenes\cbox\cbox.xml" ..
2020-03-12 07:04:00 INFO  main  [PluginManager] Loading plugin "plugins\regular.dll" ..
2020-03-12 07:04:00 INFO  main  [PluginManager] Loading plugin "plugins\srgb.dll" ..
2020-03-12 07:04:00 INFO  main  [PluginManager] Loading plugin "plugins\srgb_d65.dll" ..
2020-03-12 07:04:00 INFO  main  [PluginManager] Loading plugin "plugins\path.dll" ..
2020-03-12 07:04:00 INFO  tbb01 [PluginManager] Loading plugin "plugins\diffuse.dll" ..
2020-03-12 07:04:00 INFO  tbb02 [PluginManager] Loading plugin "plugins\independent.dll" ..
2020-03-12 07:04:00 INFO  tbb08 [PluginManager] Loading plugin "plugins\gaussian.dll" ..
2020-03-12 07:04:00 INFO  tbb10 [PluginManager] Loading plugin "plugins\obj.dll" ..
2020-03-12 07:04:00 INFO  tbb00 [PluginManager] Loading plugin "plugins\area.dll" ..
2020-03-12 07:04:00 INFO  tbb00 [PluginManager] Loading plugin "plugins\uniform.dll" ..
2020-03-12 07:04:00 INFO  tbb08 [PluginManager] Loading plugin "plugins\hdrfilm.dll" ..
2020-03-12 07:04:00 INFO  tbb02 [PluginManager] Loading plugin "plugins\perspective.dll" ..
2020-03-12 07:04:00 INFO  main  [ShapeKDTree] Building a SAH kd-tree (38 primitives) ..
2020-03-12 07:04:00 INFO  main  [ShapeKDTree] Finished. (1.53 KiB of storage, took 0ms)
Traceback (most recent call last):
  File "depth_integrator.py", line 34, in <module>
    if not sampler.ready():
AttributeError: 'mitsuba.render.Sampler' object has no attribute 'ready'
```
Sampler.ready cannot be found in the API reference.
https://mitsuba2.readthedocs.io/en/latest/generated/render_api/#sampler

So, when I commented out this part and executed it, the following new error occurred.
```
2020-03-12 07:07:38 INFO  main  [PluginManager] Loading plugin "plugins\regular.dll" ..
2020-03-12 07:07:38 INFO  main  [PluginManager] Loading plugin "plugins\srgb.dll" ..
2020-03-12 07:07:38 INFO  main  [PluginManager] Loading plugin "plugins\srgb_d65.dll" ..
2020-03-12 07:07:38 INFO  main  [PluginManager] Loading plugin "plugins\path.dll" ..
2020-03-12 07:07:38 INFO  tbb00 [PluginManager] Loading plugin "plugins\diffuse.dll" ..
2020-03-12 07:07:38 INFO  tbb00 [PluginManager] Loading plugin "plugins\area.dll" ..
2020-03-12 07:07:38 INFO  tbb04 [PluginManager] Loading plugin "plugins\independent.dll" ..
2020-03-12 07:07:38 INFO  tbb05 [PluginManager] Loading plugin "plugins\obj.dll" ..
2020-03-12 07:07:38 INFO  tbb01 [PluginManager] Loading plugin "plugins\gaussian.dll" ..
2020-03-12 07:07:38 INFO  tbb00 [PluginManager] Loading plugin "plugins\uniform.dll" ..
2020-03-12 07:07:38 INFO  tbb01 [PluginManager] Loading plugin "plugins\hdrfilm.dll" ..
2020-03-12 07:07:38 INFO  tbb04 [PluginManager] Loading plugin "plugins\perspective.dll" ..
2020-03-12 07:07:38 INFO  main  [ShapeKDTree] Building a SAH kd-tree (38 primitives) ..
2020-03-12 07:07:38 INFO  main  [ShapeKDTree] Finished. (1.53 KiB of storage, took 0ms)
Traceback (most recent call last):
  File "depth_integrator.py", line 70, in <module>
    block.put(position_sample, rays.wavelengths, Vector3f(result, result, result), 1)
NameError: name 'position_sample' is not defined
```
Indeed, the `position_sample` looks undefined.

**Environment**
- Mitsuba2 (d30def812df1b46cb0c3f605578cc6676d3dc6c3) Latest
- Windows10 Pro
- CUDA 10.1
- OptiX 6.5
- CMake 3,16.15
- Visual Studio 2019 Pro (MSVC 1924)
- Python 3.7.5

Thank you.

**Comment by Speierers** [comment]
This was fixed in 84c4d45a283ca101bfef4ee192e6b19ec53a3ed6
Thanks.

**Comment by tkuri** [comment]
Thank you for fixing bug !

**Comment by tkuri** [comment]
I'm sorry again.
I updated it to the latest version and executed it, but UInt32 is said to be undefined.

```
Traceback (most recent call last):
  File "depth_integrator.py", line 40, in <module>
    pos = ek.arange(UInt32, total_sample_count)
NameError: name 'UInt32' is not defined
```

And `position_sample` still seems undefined.

**Comment by Speierers** [comment]
Sorry for that. Should be fixed here 398a561 (this time I actually tried to run it :wink:)

**Comment by tkuri** [comment]
Thank you for fixing bug.
I was able to get the output.
